<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Postal Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePostalAddress"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="zoomSpinBox">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goButton">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="lwMarkers"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbRemoveMarker">
     <property name="text">
      <string>Remove marker</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
